examples/example-protocol/src/assets/rust_plugin_test/expected_types.rs

```diff
@@ -31,6 +31,12 @@ pub struct DocExampleStruct {
     pub r#type: String,
 }
 
+/// A point of an arbitrary type, with an explicit 'Serializable' bound.
+#[derive(Clone, Debug, Deserialize, PartialEq, Serialize)]
+pub struct ExplicitBoundPoint<T> {
```

Comment:
How come the generated version doesn't have the trait bound?

Reply:
Good question! Our `TypeIdent` struct doesn't currently store bounds for generic arguments, so I should go ahead and add the bounds there as well.